src/librustc/dep_graph/graph.rs

```diff
@@ -39,10 +39,12 @@ pub struct DepGraph {
     fingerprints: Lrc<Lock<IndexVec<DepNodeIndex, Fingerprint>>>
 }
 
-newtype_index!(DepNodeIndex);
+newtype_index! {
+    pub struct DepNodeIndex;
```

Comment:
Would be nice if you had to write `pub struct DepNodeIndex(..);` or something, to indicate there's something in there that the macro fills in!

Reply:
we could make it `pub struct DepNodeIndex { .. }` perhaps?

Reply:
I changed to `{ .. }`